herbert_core/classes/instrument_classes/instrument/detectors/@IX_detector_bank/IX_detector_bank.m

```diff
@@ -202,6 +213,25 @@
         %
         % Checks that rely on interdependencies must go here
         
+        function val = get.combined(obj)
+            val = struct();
+            val.id = obj.id_;
+            val.x2 = obj.x2_;
+            val.phi = obj.phi_;
+            val.azim = obj.azim_;
+            val.dmat = obj.dmat_;
+            val.det = obj.det_;
```

Comment:
Not sure whether this would be clearer or more obscure as:
```matlab
val = struct('id', obj.id_, 'x2', obj.x2_, ...)
```
Also, are there appropriate getters on the `obj` to not need direct access to the "private" vars?

Reply:
got the point but leaving for the moment due to Herbert merge